### PR TITLE
test: display command line options passed to send_cli() in debug log

### PR DESCRIPTION
**Pull request by jonatack**
as per https://github.com/bitcoin/bitcoin/pull/18691#discussion_r411382589, and revert two cli calls changed in #18691 from rpc commands back to command line options (these were the only occurrences).

### CODE REVIEW

**Comment by jonatack** [comment]
A good way to test this is to run `test/functional/interface_bitcoin_cli.py --loglevel debug`

File: test/functional/test_framework/test_node.py [hunk 1]
```diff
@@ -580,7 +580,8 @@ def send_cli(self, command=None, *args, **kwargs):
         if command is not None:
             p_args += [command]
         p_args += pos_args + named_args
-        self.log.debug("Running bitcoin-cli command: %s" % command)
+        options_to_log = '' if not self.options else ', options: ' + ', '.join(self.options)
+        self.log.debug("Running bitcoin-cli command: {}{}".format(command, options_to_log))
```
Comment (MarcoFalke) [inline test/functional/test_framework/test_node.py:584]: Might as well just log p_args[2:] or so, because -named is an option as well

Comment (jonatack) [reply test/functional/test_framework/test_node.py:585]: Done, set before `p_args += [command]` to not log the command in the options as well.

Comment (jonatack) [reply test/functional/test_framework/test_node.py:585]: Do you think it's more useful to set it after ` p_args += pos_args + named_args` and remove the command from the list?

Comment (MarcoFalke) [reply test/functional/test_framework/test_node.py:585]:
```py
self.log.debug("Running bitcoin-cli {}".format(p_args[2:]))
```

should do all the things:

* Include whether it is `-named` or not
* Include all args, like `--tracerpc`
* Include the command

Comment (jonatack) [reply test/functional/test_framework/test_node.py]: Done

**Comment by MarcoFalke** [comment]
ACK 8f5dc8800aeb524eee2fa2451cd22883b7b2bfec

**Pull request merged**